### PR TITLE
Create 1-Click Clusters section

### PR DESCRIPTION
**Pull request by cbrownstein-lambda**
This PR adds a 1-Click Clusters (1CC) section to the docs and seeds the section with a "Getting started" page.

This PR also adds a gitignore file to make Git ignore vim files and the directory used for local builds of the docs (`/site`).

### CODE REVIEW

**Review by pete-lambda — COMMENTED** [review]
I'm assuming the 1CC description in the first paragraph is all correct :)

File: docs/1-click-clusters/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+1-Click Clusters (1CC) are clusters of GPU and CPU instances in
+[Lambda On-Demand Cloud](https://lambdalabs.com/service/gpu-cloud) consisting
+of 16 to 512 NVIDIA H100 SXM Tensor Core GPUs. Compute (GPU) nodes are
+interconnected over an NVIDIA Quantum-2 400Gb/s InfiniBand non-blocking fabric
+in a rail-optimized topology, providing peer-to-peer GPUDirect RDMA
+communication of up to 3200Gb/s. All nodes are connected via 2x100Gb/s
+Ethernet for IP communication and are connected to the Internet via 2x100Gb/s
+Direct Internet Access (DIA) circuits.
+
+Each 1CC includes 3x CPU management (head) nodes for use as jump boxes
```
[the remark this answers is not in the document]
Comment (cbrownstein-lambda) [reply docs/1-click-clusters/getting-started.md:14]: From my experience, the terms are used interchangeably.

I think it's best to write docs with common usage of terminology in mind versus complete technical accuracy. But, if you disagree, feel free to rewrite.

File: docs/1-click-clusters/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Each compute note includes 24TB of usable local ephemeral NVMe storage. Each
+management node includes 208GB of usable local ephemeral NVMe storage.
+[Persistent storage](https://docs.lambdalabs.com/on-demand-cloud/file-systems)
+is automatically created and attached to each 1CC node, and can also be
+attached to on-demand instances. Existing file systems in the same
+region can additionally be attached.
+[You're billed only for the storage you actually use](https://docs.lambdalabs.com/on-demand-cloud/file-systems#how-are-file-systems-billed).
+
+All 1CC nodes are preinstalled with Ubuntu 22.04 LTS and
```
[the remark this answers is not in the document]
Comment (cbrownstein-lambda) [reply docs/1-click-clusters/getting-started.md:38]: Generally, we separate Ubuntu from Lambda Stack.

For example, we provide [instructions for installing Lambda Stack in Ubuntu](https://lambdalabs.com/lambda-stack-deep-learning-software).

What constitutes Lambda Stack isn't really defined.

File: docs/1-click-clusters/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (smithbrady) [reply docs/1-click-clusters/getting-started.md:14]: I don't know the architecture of 1CC but in reserved cloud the management nodes are within the network/security zone. I think that jump box is accurate for the 1CC head nodes.

[the remark this answers is not in the document]
Comment (cbrownstein-lambda) [reply docs/1-click-clusters/getting-started.md:14]: @smithbrady With RC, nodes are accessed through a VPN. That's not the case with OD.

[the remark this answers is not in the document]
Comment (pete-lambda) [reply docs/1-click-clusters/getting-started.md:14]: I'm fine with it as is; I'd never heard of a bastion host before today so I dug around a little to learn more If they're interchangeable in your experience, then that's cool by me!

File: docs/1-click-clusters/getting-started.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pete-lambda) [reply docs/1-click-clusters/getting-started.md:38]: OK sounds good; I'm still learning all this. This is good fodder for a glossary and style/usage guide!

**Pull request merged**